ixwebsocket/IXWebSocketPoll.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ *  IXWebSocketPoll.cpp
+ *  Author: Korchynskyi Dmytro
+ *  Copyright (c) 2017-2018 Machine Zone, Inc. All rights reserved.
+ */
+
+#include "IXWebSocketPoll.h"
+
+namespace ix
+{
+
```

Comment:
Could we use a shared_ptr instead of a raw pointer?
Those make me nervous.

Reply:
No, because user might not use shared_ptr

Reply:
Can we use a const reference to an IXWebSocket ?

Reply:
No, we set callback on it